Wir_sind_Eins_(Outbreakband).tex

```diff
@@ -0,0 +1,52 @@
+%&format -translate-file=tcxfile
+
+\documentclass[]{leadsheet}
+
+\begin{document}
+
+\begin{song}[transpose=7]{
+title={Wir sind eins},
+interpret={Outbreakband},
+composer={Mia Friesen | Pala Friesen},
+key={D},
+tempo={Tempo - 123 | Time - 4/4}
+}
+
+\begin{schedule}
+I -- I -- V -- R -- V -- I -- R -- Z -- B -- R -- I -- I
+\end{schedule}
+
+\begin{intro}
+| ^{Bm}\wholerest~ | ^{G}\wholerest~ | ^{D}\wholerest~ | ^{A/C#}\wholerest~ :|| \\
+\end{intro}
+
+\begin{verse}[numbered=false]
+| ^{Bm}\halfrest~ Du hast uns | \_ er - ^{G}wählt | ^{D}\halfrest~ nun sind wir | \_ dein ^{A/C#}Leib \\
+| ^{Bm}\halfrest~ Du hast uns | \_ zu - ^{G}sammenge - | ^{D}führt \halfrest~   | ^{A/C#}\wholerest~ \\
+| ^{Bm}\halfrest~ Unser Herz | \_ geht ^{G}auf | ^{D}\halfrest~ Leben strömt | \_ hi - ^{A/C#}nein \\
+| ^{Bm}\halfrest~ Du hast uns | \_zu - ^{G}sammenge - | ^{D}führt   | ^{A/C#} \halfrest~
+\end{verse}
+
+\begin{chorus}
+Wir sind | ^{Bm}eins\_\_\_ | ^{G}fol-gen dei-nem | ^{D}Namen | ^{A/C#}\halfrest~ \\
+Wir sind | ^{Bm}eins\_\_\_ | ^{G}eh-ren dich al - | ^{D}leine\_ | ^{A/C#}\halfrest~ \\
+| ^{Bm}Tren-nen kann uns kei | ^{G}- ner mehr weil | ^{D}\_ wir dir ge - hörn | ^{A/C#}\_ \quarterrest~ \\
+Du thronst | ^{Bm}\_ in Liebe un | ^{G}- ter uns und je | ^{D}- der soll es sehn | ^{A/C#}\_ \quarterrest~ \eighthrest~ \\
+Wir sind | ^{Bm}eins               | ^{A}\wholerest~
+\end{chorus}
+
+\begin {interlude}
+| ^{G}\wholerest~ | \wholerest~ | ^{A}\wholerest~ | \wholerest~
+\end{interlude}
+
+\begin{bridge}
+| ^{G}\quarterrest~ Ein Weg | \quarterrest~ ein Ziel | ^{A}\quarterrest~ ein Gott der Leben gibt \\
```

Comment:
| ^{G}\quarterrest~ Ein Weg | \quarterrest~ ein Ziel | ^{A}\quarterrest~ ein Gott der Le- | ben gibt \\
| ^{G}\quarterrest~ Ein Leib | \quarterrest~ ein Geist | ^{A}\quarterrest~ ein Gott der uns | \_ vereint \\
| ^{G}\quarterrest~ Ein Weg | \quarterrest~ ein Ziel | ^{A}\quarterrest~ ein Gott der Le- | ben gibt \\
| ^{G}\quarterrest~ Ein Leib | \quarterrest~ ein Geist | ^{A}\quarterrest~ ein Gott \quarterrest~ | \halfrest~

Reply:
Has been fixed in new commit